doc/caveats.md

```diff
@@ -0,0 +1,41 @@
+Caveats
+=======
+
+Git-dit is, obviously, git specific and will hence not work with other VCS like
+mercurial. Import and export functionality for other issue tracking systems are,
+however, not inconceivable. Functionality like referencing issues across
+repositories will, however, most likely never be provided across different VCS.
```

Comment:
If you remember some segments about "next generation" databases, one of the most prominent failures when adapting them was people not realizing functionality which they presumed would be there was missing.